### PR TITLE
Instruction to update `"bids_config"` key in microscopy tutorial

### LINKED ISSUE #1051: BIDS config path in microscopy tutorial need to be adjusted
<!-- Hi, and thank you for reporting an issue! Please take the time to first consider NeuroPoly's guidelines on issues (titles, descriptions, issue labels) before continuing:
https://www.neuro.polymtl.ca/software/contributing#opening_an_issue
-->

## Issue description
The [two-class microscopy segmentation tutorial](https://ivadomed.org/tutorials/two_class_microscopy_seg_2d_unet.html) makes the user copy the config file to their local directory via

```console
cp <PATH_TO_IVADOMED>/ivadomed/config/config_microscopy.json .
```

However, running this config file (i.e. with `ivadomed --train -c config_microscopy.json`) without any changes will result in a `ValueError` due to the fact that `ivadomed/config/config_bids.json` specified in the `"bids_config"` key of the original config file is **not a valid path**. 

### Current behavior
<!-- Please provide a brief overview of the bug or error you've encountered.
-->

The error message I get is shown below:

<details><summary>Terminal output</summary>

```console
(um_main) uzmac@romane:~/test$ ivadomed --train -c config_microscopy.json                                                                                            
2022-01-22 12:48:49.130 | INFO     | ivadomed.utils:init_ivadomed:408 -                                                                                              
ivadomed (git-master-9092196db826773205ad25047fc291e853f9fc52*)                                                                                                      
                                                                                                                                                                     
2022-01-22 12:48:49.132 | INFO     | ivadomed.utils:get_path_output:371 - CLI flag --path-output not used to specify output directory. Will check config file for dir
ectory...                                                                                                                                                            
2022-01-22 12:48:49.132 | INFO     | ivadomed.utils:get_path_data:383 - CLI flag --path-data not used to specify BIDS data directory. Will check config file for dire
ctory...                                                                                                                                                             
2022-01-22 12:48:49.132 | INFO     | ivadomed.main:set_output_path:207 - Output path already exists: log_microscopy_sem_randomcrop_try                               
2022-01-22 12:48:49.339 | INFO     | ivadomed.utils:define_device:135 - Using GPU ID 0                                                                               
2022-01-22 12:48:49.340 | INFO     | ivadomed.utils:display_selected_model_spec:145 - Selected architecture: Unet, with the following parameters:                    
2022-01-22 12:48:49.340 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   dropout_rate: 0.2                                                            
2022-01-22 12:48:49.340 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   bn_momentum: 0.1                                                             
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   depth: 4                                                                     
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   is_2d: True                                                                  
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   final_activation: sigmoid                                                    
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   folder_name: model_seg_rat_axon-myelin_sem                                   
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   in_channel: 1                                                                
2022-01-22 12:48:49.341 | INFO     | ivadomed.utils:display_selected_model_spec:148 -   out_channel: 3                                                               
Traceback (most recent call last):                                                                                                                                   
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/bin/ivadomed", line 11, in <module>                                                                                     
    load_entry_point('ivadomed', 'console_scripts', 'ivadomed')()                                                                                                    
  File "/home/GRAMES.POLYMTL.CA/uzmac/ivadomed/ivadomed/main.py", line 592, in run_main                                                                              
    run_command(context=context,                                                                                                                                     
  File "/home/GRAMES.POLYMTL.CA/uzmac/ivadomed/ivadomed/main.py", line 370, in run_command                                                                           
    bids_df = BidsDataframe(loader_params, path_output, derivatives=True)                                                                                            
  File "/home/GRAMES.POLYMTL.CA/uzmac/ivadomed/ivadomed/loader/bids_dataframe.py", line 64, in __init__                                                              
    self.create_bids_dataframe()                                                                                                                                     
  File "/home/GRAMES.POLYMTL.CA/uzmac/ivadomed/ivadomed/loader/bids_dataframe.py", line 110, in create_bids_dataframe                                                
    layout = pybids.BIDSLayout(str(path_data), config=self.bids_config, indexer=indexer,                                                                             
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/lib/python3.8/site-packages/bids/layout/layout.py", line 151, in __init__                                               
    self.connection_manager = ConnectionManager(                                                                                                                     
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/lib/python3.8/site-packages/bids/layout/db.py", line 48, in __init__                                                    
    self.reset_database(init_args, config)                                                                                                                           
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/lib/python3.8/site-packages/bids/layout/db.py", line 114, in reset_database                                             
    config = [Config.load(c, session=self.session) for c in listify(config)]                                                                                         
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/lib/python3.8/site-packages/bids/layout/db.py", line 114, in <listcomp>                                                 
    config = [Config.load(c, session=self.session) for c in listify(config)]                                                                                         
  File "/home/GRAMES.POLYMTL.CA/uzmac/.local/lib/python3.8/site-packages/bids/layout/models.py", line 160, in load                                                   
    raise ValueError("{} is not a valid path.".format(config))                                                                                                       
ValueError: ivadomed/config/config_bids.json is not a valid path.
```

</details>

Changing the `"bids_config"` key in the config file to `<PATH_TO_IVADOMED>/ivadomed/config/bids_config.json` solves this issue and I don't encounter any other issues in the rest of the tutorial.

### Expected behavior
<!-- Please provide a description of what you expected to happen instead.
--> 

We should change the tutorial s.t. we tell the user to change the `"bids_config"` key in their copied config file in their local directory. Or can we find a solution which wouldn't involve the user changing the config file 🤔?

### PR DESCRIPTION
**Pull request by uzaymacar**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
- [x] I've assigned a reviewer

#### PR contents

- [x] I've consulted [ivadomed's internal developer documentation](https://github.com/ivadomed/ivadomed/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] I've added [relevant tests](https://github.com/ivadomed/ivadomed/wiki/tests) for my contribution
- [x] I've updated the [relevant documentation](https://github.com/ivadomed/ivadomed/wiki/documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description

This PR adds an instruction to the [two-class microscopy segmentation tutorial](https://ivadomed.org/tutorials/two_class_microscopy_seg_2d_unet.html) for the user to update the `"bids_config"` key in the copied config file in their local directory.

## Linked issues

Resolves #1051.

### CODE REVIEW

**Review by mariehbourget — COMMENTED** [review]
Thank you @uzaymacar for opening this PR to fix the microscopy tutorial.

I agree with your fix but I would move the instructions to change the `bids_config` value directly under its specific section.
Let me know what you think.

File: docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst [hunk 1]
```diff
@@ -34,7 +34,11 @@ First off, copy this configuration file in your local directory (to avoid modify
 
    cp <PATH_TO_IVADOMED>/ivadomed/config/config_microscopy.json .
 
-Then, open it with a text editor. Below we will discuss some of the key parameters to perform a two-class 2D
+Then, open it with a text editor. You will need to update the value of the
+``loader_parameters:bids_config`` key from ``"ivadomed/config/config_bids.json"`` to
+``"<PATH_TO_IVADOMED>/ivadomed/config/config_bids.json"``.
+
```
[the remark this answers is not in the document]
Comment (uzaymacar) [reply docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst:40]: I'm OK either way but just to clarify, I don't think path data needs to be changed. After downloading the data with `ivadomed_download_data -d data_axondeepseg_sem`, I didn't have to change anything else.

File: docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst [hunk 2]
```diff
@@ -61,7 +65,7 @@ microscopy segmentation training.
```
[the remark this answers is not in the document]
Comment (mariehbourget) [reply docs/source/tutorials/two_class_microscopy_seg_2d_unet.rst:65]: Yes, good idea, a Note type box would certainly help

**Review by mariehbourget — APPROVED** [review]
Looks good! Thank you @uzaymacar!

**Pull request merged**